frontend/src/components/App/index.js

```diff
@@ -16,14 +17,16 @@ class App extends React.Component {
   render() {
     return (
       <AuthProvider>
-        <Router>
-          <div>
-            <Route exact path={ROUTES.LANDING} component={LandingPage} />
-            <Route path={ROUTES.SIGN_IN} component={SignInPage} />
-            <PrivateRoute path={ROUTES.VIEW_TRIPS} component={ViewTripsPage} />
-            <PrivateRoute path={ROUTES.VIEW_ACTIVITIES} component={ViewActivitiesPage} />
-          </div>
-        </Router>
+        <AuthUtilsConsumer>
+          <Router>
+            <div>
+              <Route exact path={ROUTES.LANDING} component={LandingPage} />
+              <Route path={ROUTES.SIGN_IN} component={SignInPage} />
+              <PrivateRoute path={ROUTES.VIEW_TRIPS} component={ViewTripsPage} />
+              <PrivateRoute path={ROUTES.VIEW_ACTIVITIES + '/:tripId'} component={ViewActivitiesPage} />
```

Comment:
I think @zghera  mentioned somewhere that the :tripID is no longer necessary. sorry 'bout any confusion.